packages/calcite-components/src/assets/styles/_x-button.scss

```diff
@@ -0,0 +1,59 @@
+@mixin xButton(
+  $textColor: "--calcite-color-text-3",
+  $backgroundColor: "--calcite-color-foreground-2",
+  $hoverTextColor: "--calcite-color-text-3",
+  $hoverBackgroundColor: "--calcite-color-foreground-3",
+  $activeBorderColor: "--calcite-color-brand",
+  $borderWidth: "--calcite-border-width-md"
+) {
+  .x-button {
+    appearance: none;
```

Comment:
I thought we agreed to only inline Tailwind utils related to component tokens. FWIW, we can automate this step later to make it easier for review.